index.js

```diff
@@ -0,0 +1,377 @@
+const express = require('express'); // pull in express
+const cors = require('cors');
+const server = express(); // instantiate server
+const logger = require('morgan');
+const db = require('./data/helpers/userDb');
+const postDb = require('./data/helpers/postDb');
+const port = 5201;
+
+
+const capitalize = (req, res, next) => {
+  const { name } = req.body;
+  req.body.name = req.body.name.charAt(0).toUpperCase() + req.body.name.slice(1);
+  next();
+};
+
+const textRecognize = (req, res, next) => {
+  const newText = req.body.text;
+  req.text = newText
+};
+
+
+server.use(cors()); // connects react
```

Comment:
Just to clarify, cors isn't necessarily to "connect React". What it does is allows us to accept requests from different domains (other than the server's). So while it will allow us to interact from the front end, that does not strictly mean it 'connects React'.